politeiawww/eventmanager.go

```diff
@@ -372,58 +368,57 @@ func (p *politeiawww) notifyProposalAuthorOnComment(d dataProposalComment) error
 	// Send notification eamil
 	commentID := strconv.FormatUint(uint64(d.commentID), 10)
 	return p.emailProposalCommentSubmitted(d.token, commentID, d.username,
-		proposalName(*pr), author.Email)
+		proposalName(pr), author.Email)
 }
 
-func (p *politeiawww) notifyParentAuthorOnComment(d dataProposalComment) error {
+func (p *politeiawww) notifyParentAuthorOnComment(d dataProposalComment, pr pi.ProposalRecord) error {
 	// Verify this is a reply comment
 	if d.parentID == 0 {
 		return nil
 	}
 
 	// Lookup the parent comment author to check if they should receive
 	// a reply notification.
+	parentComment, err := p.commentsGet(comments.Get{
+		State:      convertCommentsStateFromPi(d.state),
+		Token:      d.token,
+		CommentIDs: []uint32{d.parentID},
+	})
+	if err != nil {
+		return err
+	}
+	userID, err := uuid.Parse(parentComment.Comments[0].UserID)
+	if err != nil {
+		return err
+	}
+	author, err := p.db.UserGetById(userID)
+	if err != nil {
+		return err
+	}
 
-	// Get the parent comment
-	// TODO
-	return nil
-
-	/*
-		// Lookup the parent comment author
-		var author *user.User
-
-		// Check if notification should be sent to author
-		switch {
-		case d.username == author.Username:
-			// Author commented on their own proposal
-			return nil
-		case !userNotificationEnabled(*author,
-			www.NotificationEmailCommentOnMyProposal):
-			// Author does not have notification bit set on
-			return nil
-		}
-
-		// Get proposal. We need this proposal name for the notification.
-		pr, err := p.proposalRecordLatest(d.state, d.token)
-		if err != nil {
-			return fmt.Errorf("proposalRecordLatest %v %v: %v",
-				d.state, d.token, err)
-		}
-
-		// Send notification eamil
-		commentID := strconv.FormatUint(uint64(d.commentID), 10)
+	// Check if notification should be sent
+	switch {
+	case d.username == author.Username:
+		// Author replied to their own comment
+		return nil
+	case !userNotificationEnabled(*author,
+		www.NotificationEmailCommentOnMyComment):
+		// Author does not have notification bit set on
+		return nil
+	}
 
-		return p.emailProposalCommentReply(d.token, commentID, d.username,
-			proposalName(*pr), author.Email)
-	*/
+	// Send notification email to parent comment author
+	commentID := strconv.FormatUint(uint64(d.commentID), 10)
+	return p.emailProposalCommentReply(d.token, commentID, d.username,
+		proposalName(pr), author.Email)
 }
 
 type dataProposalComment struct {
-	state     pi.PropStateT
-	token     string
-	commentID uint32
-	parentID  uint32
-	username  string // Comment author username
+	state     pi.PropStateT // Proposal state
```

Comment:
I left these comments out because they're redundant.  They don't add any additional information that the field name doesn't already give.